### PR TITLE
Update /become-a-partner

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Updated the [/become-a-partner](https://canonical-com-887.demos.haus/partners/become-a-partner) page according to [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9427-18076&t=2gCmy0Ck7MrzwKzA-0)
- Removed outdated `is-paper` class
- Formatted `styles.scss`

## QA

- Compare [/become-a-partner](https://canonical-com-887.demos.haus/partners/become-a-partner) against [the design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9427-18076&t=2gCmy0Ck7MrzwKzA-0)
- Make sure that it looks correct and nothing is broken

## Issue / Card

Fixes WD-3745

## Before merging
If the image is the final one we want to use we should add it to the asset manager instead of hotlinking.

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -638,7 +631,31 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
...
-      width: 121px;  
+      width: 121px;
+    }
+  }
+}
+
+.p-overlayed-section {
+  display: grid;
+  grid-template-columns: 2fr 8fr;
+  grid-template-rows: 477px 106px 1fr;
```
Comment (lyubomir-popov) [inline static/sass/styles.scss:642]: This feels very fragile, and won't update when the value of 1rem changes after 1680px. Can this be done with just a regular strip wrapping this and then using the variable in the regular strip for an overlap value?

**Review by lyubomir-popov — CHANGES_REQUESTED** [review]
A couiple of comments in figma https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?type=design&node-id=9960%3A24523&t=iDY29WLvf5WCb1hc-1

File: templates/partners/become-a-partner.html [hunk 2]
```diff
@@ -6,408 +6,346 @@
...
+    <hr class="p-rule">
+  </div>
+  <div class="row--25-75">
+    <div class="col">
+      <div class="p-block">
+        <h2>How we work with our partners</h2>
       </div>
-      <p>
-        We run a range of partner programmes to help you maximise revenue and efficiency from the opportunities Ubuntu provides.
+      <p>By formalising their partnership with Canonical, technology companies of all kinds –
```
Comment (mtruj013) [inline templates/partners/become-a-partner.html:39]:
```suggestion
      <p>By formalising their partnership with Canonical, technology companies of all kinds &ndash; 
```

File: templates/partners/become-a-partner.html [hunk 3]
```diff
@@ -6,408 +6,346 @@
...
+  </div>
+  <div class="row--25-75">
+    <div class="col">
+      <div class="p-block">
+        <h2>How we work with our partners</h2>
       </div>
-      <p>
-        We run a range of partner programmes to help you maximise revenue and efficiency from the opportunities Ubuntu provides.
+      <p>By formalising their partnership with Canonical, technology companies of all kinds – 
+from hardware makers and ISVs to cloud providers and systems integrators – benefit from significant opportunities to increase their revenues with Ubuntu.
```
Comment (mtruj013) [inline templates/partners/become-a-partner.html:40]:
```suggestion
from hardware makers and ISVs to cloud providers and systems integrators &ndash; benefit from significant opportunities to increase their revenues with Ubuntu.
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just a couple super minor comments. Also not sure if the split is supposed to be 50/50 on medium screens like we've been doing for new pages on u.com, @lyubomir-popov could you confirm?

**Review by lyubomir-popov — APPROVED** [review]
I pushed the changes I'd like to see, so giving a +1.
@ClementChaumel please have a look at what I've done code-wise:, the grid refactor, the overlap calculations in particular.


image
image

**Comment by lyubomir-popov** [comment]
> layout? And maybe name it + explain in which cases it should be used?
> And generally, could you expand on how each layout's (50%-50%, 25%-75%, etc) responsiveness?

Yes we discussed this with @bartaz today in the standup. We'll have a short session to describe the variants we need tomorrow and we'll document the decisions.

**Pull request merged**